Spanner/tests/Unit/DatabaseTest.php

```diff
@@ -839,7 +839,7 @@ public function testRunTransactionRetry()
 
         $this->refreshOperation($this->database, $this->connection->reveal());
 
-        $this->database->runTransaction(function ($t) use ($it) {
+        $this->database->runTransaction(function ($t) use (&$it) {
```

Comment:
Wasn't this a bug not sending the 'isRetry' status?
Ref: https://cloud.google.com/php/docs/reference/cloud-spanner/latest/Transaction#_Google_Cloud_Spanner_Transaction__isRetry__